qa/full-cluster-restart/src/test/java/org/elasticsearch/upgrades/FullClusterRestartIT.java

```diff
@@ -152,10 +157,129 @@ void assertBasicSearchWorks(String index) throws IOException {
         assertEquals(numDocs1, numDocs2);
     }
 
+    void assertAllSearchWorks(String index) throws IOException {
+        logger.info("--> testing _all search");
+        Map<String, Object>  searchRsp = toMap(client().performRequest("GET", "/" + index + "/_search"));
+        assertNoFailures(searchRsp);
+        int totalHits = (int) XContentMapValues.extractValue("hits.total", searchRsp);
+        assertTrue("totalHits < " + totalHits, totalHits >=  1);
```

Comment:
sure, on the other hand I try to use junit's `Assert.*` methods where possible. I'm not a big fan of using both test libraries in the same test case. But in this case it does improve readability.